id,created_at,closed_at,state,comments,title
661,2020-08-30 10:09:08.000000000,1970-01-01 00:00:00.000000001,open,0,[QUESTION] How to use --nofail and --return-success
660,2020-08-28 23:08:08.000000000,2020-08-28 23:33:38.000000000,closed,0,Google sync
659,2020-08-28 03:50:39.000000000,1970-01-01 00:00:00.000000001,open,0,pytype does not pring 'leaving directory' - affects emacs
658,2020-08-27 04:48:57.000000000,2020-08-28 22:50:18.000000000,closed,0,Google sync
657,2020-08-27 03:15:59.000000000,2020-08-27 03:48:58.000000000,closed,0,Clean up some lint errors.
656,2020-08-27 03:09:34.000000000,1970-01-01 00:00:00.000000001,open,0,"Enable lint warnings super-with-arguments, raise-missing-from"
655,2020-08-27 01:23:51.000000000,2020-08-27 01:34:00.000000000,closed,7,[QUESTION] Use Pytype for static type inference of local variables. 
654,2020-08-25 15:45:23.000000000,1970-01-01 00:00:00.000000001,open,1,Inheritance Method Resolution Order
653,2020-08-24 02:24:43.000000000,1970-01-01 00:00:00.000000001,open,3,`or` doesn't figure out that an `optional` on left hand side evaluates to right hand side
652,2020-08-23 02:12:59.000000000,1970-01-01 00:00:00.000000001,open,1,attribute-error ignores parent variable annotations
651,2020-08-22 05:10:31.000000000,1970-01-01 00:00:00.000000001,open,4,Pytype gives wrong type for optional attributes in dataclasses
650,2020-08-21 17:39:32.000000000,1970-01-01 00:00:00.000000001,open,1,Insufficient handling of memoryview for re ... and elsewhere
649,2020-08-21 04:40:02.000000000,2020-08-21 05:12:14.000000000,closed,0,Google sync
648,2020-08-18 19:52:04.000000000,2020-08-19 03:42:27.000000000,closed,2,How can I exclude a directory from pytype checking?
647,2020-08-18 04:18:04.000000000,2020-08-18 04:52:05.000000000,closed,0,Support TypedDict function-based syntax in pyi files.
646,2020-08-18 01:05:46.000000000,2020-08-18 02:09:17.000000000,closed,0,Google sync
645,2020-08-17 22:13:29.000000000,1970-01-01 00:00:00.000000001,open,1,Configure with pyproject.toml
644,2020-08-17 16:40:57.000000000,2020-08-18 02:09:18.000000000,closed,2,type ignore comment after method aliasing doesn't work
643,2020-08-10 23:38:47.000000000,2020-08-10 23:39:02.000000000,closed,0,Google sync
642,2020-08-10 04:20:45.000000000,2020-08-22 22:59:06.000000000,closed,6,No such file or directory: 'ninja'
641,2020-08-06 19:07:22.000000000,2020-08-06 19:11:53.000000000,closed,1,`attribute-error` false positive on iteration
640,2020-08-05 23:09:20.000000000,2020-08-11 22:47:06.000000000,closed,5,`annotation-type-mismatch` reports false positive
639,2020-08-05 01:43:57.000000000,2020-08-05 02:08:01.000000000,closed,0,Google sync
638,2020-07-31 16:01:04.000000000,1970-01-01 00:00:00.000000001,open,1,Problem handling union types
637,2020-07-31 13:33:06.000000000,2020-08-18 04:52:05.000000000,closed,1,creating TypedDicts with TypedDict(...)
636,2020-07-31 03:24:11.000000000,2020-07-31 04:08:37.000000000,closed,0,Do a PyPI release.
635,2020-07-31 02:17:27.000000000,2020-07-31 03:03:53.000000000,closed,0,"When raising not-supported-yet for Alias = Union[T, ...] set the typeâ€¦"
634,2020-07-30 03:51:32.000000000,2020-07-30 04:56:35.000000000,closed,5,Machine readable output
633,2020-07-30 00:46:54.000000000,2020-07-30 02:16:21.000000000,closed,0,Make imports_map_loader_test work with custom $TMPDIR values.
632,2020-07-29 22:15:10.000000000,2020-07-29 23:25:41.000000000,closed,0,Google sync
631,2020-07-29 19:27:54.000000000,1970-01-01 00:00:00.000000001,open,2,"async def given `coroutine` type, but requires the inner return type?"
630,2020-07-29 16:26:12.000000000,1970-01-01 00:00:00.000000001,open,0,Inversed dict annotation for dataclasses and comprehensions (python3.8)?
629,2020-07-26 22:36:17.000000000,2020-07-30 02:16:21.000000000,closed,1,"imports_map_loader_test relies on $TMPDIR = ""/tmp"""
628,2020-07-25 23:46:45.000000000,1970-01-01 00:00:00.000000001,open,1,Inheritance from dataclass across modules deduces wrong __init__
627,2020-07-25 01:49:36.000000000,2020-07-25 02:15:17.000000000,closed,0,Google sync
626,2020-07-25 01:04:24.000000000,2020-07-31 04:24:07.000000000,closed,4,crash with two files importing each other: AttributeError: 'UnionType' object has no attribute 'cls'
625,2020-07-21 07:05:49.000000000,2020-07-21 07:28:13.000000000,closed,0,Remove no-longer-necessary docker installation step.
624,2020-07-21 01:13:06.000000000,2020-07-21 01:30:18.000000000,closed,0,Google sync
623,2020-07-19 09:24:45.000000000,2020-07-21 04:32:16.000000000,closed,5,Inference on metaclasses
622,2020-07-15 17:28:25.000000000,1970-01-01 00:00:00.000000001,open,1,Pytype fails to understand that typing.Union[<length-one-tuple>] is just <the one element of that length-one-tuple>
621,2020-07-15 06:21:28.000000000,2020-07-15 22:50:41.000000000,closed,0,Add --jobs=auto to the travis script's pytype run.
620,2020-07-15 01:52:34.000000000,2020-07-15 04:58:44.000000000,closed,0,Google sync
619,2020-07-10 17:39:46.000000000,1970-01-01 00:00:00.000000001,open,2,Classes defined inside a function aren't recognised as valid type annotations
618,2020-07-09 03:14:35.000000000,2020-07-09 03:39:19.000000000,closed,0,Google sync
617,2020-07-08 15:51:02.000000000,1970-01-01 00:00:00.000000001,open,1,Python 3.8: Pytype fails when returning tuple or list within a contextmanager scope
616,2020-07-02 17:30:43.000000000,1970-01-01 00:00:00.000000001,open,2,Pytype seems confused by aliasing in some cases
615,2020-07-01 12:15:54.000000000,2020-07-02 01:22:10.000000000,closed,2,Wrong handling of defaultdict since 2020.06.26
614,2020-06-30 03:47:04.000000000,2020-06-30 04:18:59.000000000,closed,0,Google sync
613,2020-06-28 14:22:22.000000000,2020-07-21 09:25:32.000000000,closed,2,TypedDict and total=false
612,2020-06-27 00:34:52.000000000,2020-07-25 03:18:01.000000000,closed,8,pytype hangs on alias from another module inside class
611,2020-06-27 00:27:24.000000000,1970-01-01 00:00:00.000000001,open,2,Inner class can't be used as type
610,2020-06-27 00:21:23.000000000,1970-01-01 00:00:00.000000001,open,2,Unindented comment inside class confuses parser
609,2020-06-26 23:53:08.000000000,2020-06-27 00:40:00.000000000,closed,0,Google sync
608,2020-06-26 18:54:21.000000000,2020-07-02 03:12:30.000000000,closed,2,New Release for fixes required for typeshed's Big Reformat?
607,2020-06-24 02:24:51.000000000,2020-06-24 02:25:11.000000000,closed,0,Google sync
606,2020-06-21 16:10:21.000000000,2020-06-22 23:39:52.000000000,closed,1,pytype-all stops after first file with errors
605,2020-06-21 02:02:20.000000000,2020-06-24 02:25:12.000000000,closed,1,Autodetect number of jobs with --jobs auto
604,2020-06-21 01:53:16.000000000,2020-06-24 02:25:12.000000000,closed,0,Replace list comprehension with generator expresion when calling any()
603,2020-06-20 13:34:42.000000000,2020-06-22 23:47:52.000000000,closed,1,Incorrect type if func name is same as module name
602,2020-06-16 17:48:07.000000000,2020-06-17 02:57:10.000000000,closed,1,attribute-error on attributes inherited from multiprocessing.Process
601,2020-06-16 09:00:12.000000000,2020-06-17 03:21:54.000000000,closed,2,pytype ignores bad-return-type error for matplotlib class
600,2020-06-13 02:24:35.000000000,2020-06-13 03:59:49.000000000,closed,0,Google sync
599,2020-06-02 01:20:01.000000000,2020-06-02 02:00:28.000000000,closed,0,Update __version__ and CHANGELOG for a PyPI release.
598,2020-06-01 22:56:18.000000000,2020-06-02 00:09:39.000000000,closed,0,Google sync
597,2020-05-29 01:40:59.000000000,1970-01-01 00:00:00.000000001,open,1,Add support for NewType in stubs
596,2020-05-28 01:54:11.000000000,2020-05-28 03:21:33.000000000,closed,0,Google sync
595,2020-05-27 07:31:13.000000000,2020-05-27 09:06:33.000000000,closed,1,Update listing of Python magic numbers/bytes
594,2020-05-26 00:20:34.000000000,2020-05-26 23:20:44.000000000,closed,0,Ignore cmake-generated files
593,2020-05-20 23:31:23.000000000,2020-07-02 02:07:05.000000000,closed,10,Wheels
592,2020-05-19 03:35:09.000000000,2020-05-19 04:05:02.000000000,closed,0,Google sync
591,2020-05-19 03:11:24.000000000,2020-05-19 04:31:40.000000000,closed,0,Update typeshed to 5fe6a5ba8b5d362f9339d5fef4d31ba1704c8924.
590,2020-05-18 10:05:55.000000000,1970-01-01 00:00:00.000000001,open,2,incorrectly flagged [attribute-error] for telnetlib.Telnet
589,2020-05-15 14:09:44.000000000,1970-01-01 00:00:00.000000001,open,1,pytype silent when enum member given instead of built-in type
588,2020-05-15 04:12:56.000000000,2020-05-15 04:39:01.000000000,closed,0,Google sync
587,2020-05-14 11:30:57.000000000,2020-06-02 00:09:40.000000000,closed,10,Add minimum changes required for py3.8 support
586,2020-05-14 01:31:40.000000000,2020-05-14 02:08:25.000000000,closed,0,Prepare for a PyPI release.
585,2020-05-13 14:28:21.000000000,2020-05-14 01:04:01.000000000,closed,0,Add a top-level .gitignore
584,2020-05-12 03:18:23.000000000,2020-05-12 03:44:52.000000000,closed,0,Google sync
583,2020-05-11 20:51:20.000000000,2020-05-12 04:13:10.000000000,closed,1,Dumping inferred type annotations
582,2020-05-08 00:08:59.000000000,2020-05-08 00:33:02.000000000,closed,0,Google sync
581,2020-05-05 17:21:02.000000000,1970-01-01 00:00:00.000000001,open,1,False positive due to ignored None check
580,2020-05-05 17:10:02.000000000,1970-01-01 00:00:00.000000001,open,2,False positive for disjoint type ranges
579,2020-05-05 08:20:58.000000000,2020-05-05 08:47:21.000000000,closed,0,Google sync
578,2020-05-04 11:53:50.000000000,1970-01-01 00:00:00.000000001,open,3,sqlalchemy stubs support
577,2020-05-01 02:01:46.000000000,2020-05-01 02:32:42.000000000,closed,0,Google sync
576,2020-04-28 02:51:39.000000000,2020-04-28 03:18:12.000000000,closed,0,Use dir() instead of __dict__ to find test methods.
575,2020-04-28 02:21:48.000000000,2020-04-28 02:42:33.000000000,closed,0,Google sync
574,2020-04-27 09:58:38.000000000,2020-05-19 04:36:43.000000000,closed,3,codecs.CodecInfo not type checked correctly due to a Callable not matching a callback protocol
573,2020-04-26 08:05:59.000000000,1970-01-01 00:00:00.000000001,open,2,pytype can't handle attr classes with attr.s(auto_attribs=True)
572,2020-04-24 19:32:02.000000000,2020-04-28 02:42:35.000000000,closed,0,Two inner dataclasses with same name not handled correctly
571,2020-04-24 16:58:12.000000000,2020-04-28 02:42:34.000000000,closed,0,NumPy @ operator not handled correctly
570,2020-04-24 02:03:40.000000000,1970-01-01 00:00:00.000000001,open,10,Coordinate CONTRIBUTING.md and our new dev docs
569,2020-04-24 01:58:53.000000000,2020-04-26 21:09:42.000000000,closed,2,Add a requirements.txt.
568,2020-04-23 22:23:58.000000000,1970-01-01 00:00:00.000000001,open,0,pyi parser does not support all variants of __all__
567,2020-04-23 02:59:11.000000000,2020-04-23 03:48:35.000000000,closed,0,Google sync
566,2020-04-22 04:13:54.000000000,2020-04-22 05:31:49.000000000,closed,0,Google sync
565,2020-04-21 02:29:51.000000000,2020-04-21 02:49:30.000000000,closed,0,Google sync
564,2020-04-18 05:09:55.000000000,2020-04-18 05:28:52.000000000,closed,0,Support skipping an entire TestCase.
563,2020-04-16 23:34:01.000000000,2020-04-21 02:49:31.000000000,closed,3,Strays 'nothing' type for empty tuple inference.
562,2020-04-16 04:18:48.000000000,2020-07-16 04:05:04.000000000,closed,5,set up a build system for pytype binary wheels
561,2020-04-15 02:36:37.000000000,2020-04-18 07:30:26.000000000,closed,1,"Pytype does not understand inheritance from `typing` types, but understands aliasing"
560,2020-04-11 04:17:59.000000000,2020-04-11 23:32:20.000000000,closed,0,Google sync
559,2020-04-09 22:40:06.000000000,2020-05-15 03:07:26.000000000,closed,5,Initial stab at Python 3.8 support
558,2020-04-09 22:27:54.000000000,2020-04-10 00:46:52.000000000,closed,3,Test against 3.8 and master with allowed failures.
557,2020-04-09 15:47:32.000000000,2020-04-09 22:14:31.000000000,closed,3,Failure while running tests.
556,2020-04-09 11:27:24.000000000,2020-04-11 23:32:22.000000000,closed,5,Skips built in functions while merging pyi.
555,2020-04-09 11:24:41.000000000,2020-04-09 11:25:08.000000000,closed,1,Skips built in functions while merging pyi.
554,2020-04-08 02:19:23.000000000,2020-04-09 03:22:48.000000000,closed,1,Call sys.exit() with the return code in build_scripts/pyexe.
553,2020-04-08 02:01:12.000000000,2020-04-08 02:01:21.000000000,closed,0,Google sync
552,2020-04-07 23:11:03.000000000,2020-04-08 01:14:06.000000000,closed,2,Drop host Python 2 support.
551,2020-04-07 03:35:06.000000000,2020-04-08 02:50:17.000000000,closed,2,Does pytype support deferred typing imports?
550,2020-04-05 21:13:33.000000000,2020-04-11 23:32:22.000000000,closed,7,merge-pyi feature request: do not annotate builtin functions
549,2020-04-05 21:04:48.000000000,2020-04-21 02:49:31.000000000,closed,5,Stray 'nothing' type for empty tuple inference
548,2020-04-05 20:24:25.000000000,1970-01-01 00:00:00.000000001,open,3,Typing failure when assigning function from other modules to method
547,2020-04-04 18:09:08.000000000,2020-04-06 22:19:43.000000000,closed,1,self-check: Graph contains a cycle or graph changed during iteration
546,2020-04-04 00:29:39.000000000,1970-01-01 00:00:00.000000001,open,3,wrong inferred return type for coroutine
545,2020-04-03 23:13:31.000000000,2020-04-23 10:44:10.000000000,closed,1,Dropping host Python 2 support
544,2020-04-02 19:40:28.000000000,2020-04-06 22:02:03.000000000,closed,4,"Install fails with python3.6 on WSL, python3.7 on Ubuntu 18.0.4"
543,2020-04-02 04:41:10.000000000,2020-04-02 05:08:49.000000000,closed,0,Google sync
542,2020-03-31 23:46:48.000000000,2020-04-02 00:25:11.000000000,closed,1,Disable lint error not-an-iterable.
541,2020-03-29 01:58:33.000000000,2020-04-02 05:08:50.000000000,closed,3,Note that install is broken on Alpine Linux
540,2020-03-28 17:16:38.000000000,2020-03-31 00:10:11.000000000,closed,7,Fix for type coroutine
539,2020-03-28 02:47:45.000000000,2020-03-28 03:39:13.000000000,closed,0,Google sync
538,2020-03-28 00:23:53.000000000,2020-03-28 03:49:05.000000000,closed,4,Pip install failing (package directory 'find:' does not exist)
537,2020-03-25 07:38:39.000000000,1970-01-01 00:00:00.000000001,open,2,Support for input on stdin?
536,2020-03-23 22:22:52.000000000,2020-03-23 23:02:01.000000000,closed,0,Google sync
535,2020-03-21 09:23:41.000000000,1970-01-01 00:00:00.000000001,open,2,"Generic, ABC / ABCMeta and __getitem__ can't work together"
534,2020-03-21 01:48:34.000000000,2020-03-23 22:17:23.000000000,closed,1,Google sync
533,2020-03-20 19:07:26.000000000,1970-01-01 00:00:00.000000001,open,1,Attribute Error with Union and isinstance
532,2020-03-20 02:29:04.000000000,2020-03-20 03:02:28.000000000,closed,0,Google sync
531,2020-03-12 23:14:42.000000000,2020-03-16 22:57:30.000000000,closed,3,Any recommendations for running pytype with a linter?
530,2020-03-12 05:00:26.000000000,2020-03-12 05:38:26.000000000,closed,0,Google sync
529,2020-03-11 01:37:33.000000000,2020-03-11 02:47:40.000000000,closed,0,Google sync
528,2020-03-09 16:49:07.000000000,2020-03-09 20:14:46.000000000,closed,1,static method returning an instance of its class gives typing errors; works fine in python itself and mypy
527,2020-03-09 16:24:26.000000000,1970-01-01 00:00:00.000000001,open,1,Deduction error on the type of async function
526,2020-03-07 09:13:37.000000000,2020-03-10 00:43:09.000000000,closed,4,No _SupportsIndex in module __builtin__
525,2020-03-06 21:25:27.000000000,2020-03-06 21:57:42.000000000,closed,4,How to run pytype in WSL
524,2020-03-06 09:44:20.000000000,1970-01-01 00:00:00.000000001,open,1,Protocol with variables fails on wrong-arg-types
523,2020-03-06 09:39:47.000000000,1970-01-01 00:00:00.000000001,open,1,Empty Protocol fails with wrong-arg-types
522,2020-03-04 23:14:26.000000000,2020-03-06 21:24:20.000000000,closed,5,ninja: error: build.ninja:7: expected build command name
521,2020-03-04 18:11:46.000000000,1970-01-01 00:00:00.000000001,open,1,Support typing.Literal in source code.
520,2020-03-04 00:21:38.000000000,2020-03-04 03:19:33.000000000,closed,0,Google sync
519,2020-03-01 12:26:13.000000000,2020-03-04 03:19:35.000000000,closed,0,"Add lexing for base 2, 8, and 16 literals"
518,2020-02-26 06:14:58.000000000,2020-02-26 06:45:44.000000000,closed,0,Google sync
517,2020-02-21 02:57:18.000000000,2020-02-21 03:36:56.000000000,closed,0,Do a PyPI release of the implicit Optional change.
516,2020-02-21 02:41:55.000000000,1970-01-01 00:00:00.000000001,open,3,"Protocol definition with ellipsis rejected as ""bad-return-type"""
515,2020-02-19 22:33:47.000000000,2020-02-19 23:33:17.000000000,closed,0,Google sync
514,2020-02-15 06:13:51.000000000,2020-02-18 22:39:23.000000000,closed,0,Allow a range to be sliced in Python 3.
513,2020-02-14 15:53:59.000000000,2020-02-18 22:39:23.000000000,closed,4,Pytype seems get wrong builtins range definition
512,2020-02-13 00:29:25.000000000,2020-02-13 01:58:13.000000000,closed,0,Remove pytype's support for implicit None annotations.
511,2020-02-12 06:40:31.000000000,1970-01-01 00:00:00.000000001,open,1,invalid-annotation for TypeVar in inner function
510,2020-02-07 04:39:37.000000000,2020-02-07 05:07:21.000000000,closed,0,Add __init__ files to two tools modules (#508)
509,2020-02-06 21:54:10.000000000,2020-02-11 02:52:55.000000000,closed,5,Further modernize python build system
508,2020-02-06 21:33:23.000000000,2020-02-07 05:07:20.000000000,closed,3,Add __init__ files to two tools modules
507,2020-02-06 20:07:26.000000000,2020-02-11 02:23:05.000000000,closed,2,Improvements to setup.py
506,2020-02-06 06:58:10.000000000,2020-02-06 09:36:11.000000000,closed,0,Google sync
505,2020-01-31 17:38:30.000000000,2020-05-15 16:29:56.000000000,closed,4,module-attr errors on name collisions
504,2020-01-30 20:46:56.000000000,1970-01-01 00:00:00.000000001,open,2,`pytype` breaks when importing from `tensorflow`
503,2020-01-29 21:07:59.000000000,2020-01-29 23:52:34.000000000,closed,1,utcoffset error
502,2020-01-29 04:07:30.000000000,2020-01-29 04:37:50.000000000,closed,0,Update typeshed to d5851eca6f448b648e1a7b3f3010cc3dde498535.
501,2020-01-28 18:33:15.000000000,2020-01-29 00:18:22.000000000,closed,2,Ubuntu: Trouble installing virtualenv; can't find skbuild.
500,2020-01-27 15:21:04.000000000,1970-01-01 00:00:00.000000001,open,2,__new__ in tuple subclasses
499,2020-01-25 23:54:31.000000000,1970-01-01 00:00:00.000000001,open,3,Error of paths with spaces (Linux)
498,2020-01-25 01:03:20.000000000,2020-01-25 03:36:13.000000000,closed,0,Google sync
497,2020-01-23 23:38:37.000000000,2020-01-24 00:49:36.000000000,closed,0,Google sync
496,2020-01-22 02:16:54.000000000,2020-01-24 00:49:38.000000000,closed,3,Supporting multiple disabled rules in .cfg files
495,2020-01-21 22:16:31.000000000,2020-01-24 22:40:41.000000000,closed,10,Can't install on Python 2.7 / 3.6+
494,2020-01-21 07:37:13.000000000,2020-03-04 03:19:35.000000000,closed,2,pytype can't parse hex literals
493,2020-01-18 09:00:57.000000000,1970-01-01 00:00:00.000000001,open,1,False positive `str.isascii` missing attribute
492,2020-01-11 22:26:05.000000000,2020-04-22 02:25:41.000000000,closed,2,ClassVar doesn't passthrough to its underlying type.
491,2020-01-10 00:57:08.000000000,1970-01-01 00:00:00.000000001,open,1,isinstance confusion on classes that have the Any type
490,2020-01-09 03:13:37.000000000,2020-01-09 04:35:53.000000000,closed,0,Do another PyPI release for a typeshed bug fix.
489,2020-01-08 03:14:46.000000000,2020-01-08 03:41:58.000000000,closed,0,Google sync
488,2020-01-08 00:05:29.000000000,1970-01-01 00:00:00.000000001,open,4,Support builtins.object
487,2020-01-07 22:45:32.000000000,2020-01-07 23:28:46.000000000,closed,0,Update pytype CHANGELOG and __version__ for PyPI release.
486,2020-01-07 21:03:56.000000000,2020-01-08 03:41:59.000000000,closed,1,Confusing error for a correct python snippet
485,2020-01-07 07:38:58.000000000,2020-01-24 00:49:37.000000000,closed,2,"Don't fail on mypy-style '# type: ignore[code, ...]' comments"
484,2020-01-06 23:42:40.000000000,1970-01-01 00:00:00.000000001,open,2,Really long / unterminating analysis on bm_regex_v8.py
483,2020-01-05 23:10:28.000000000,1970-01-01 00:00:00.000000001,open,1,Support custom pyi for third-party deps
482,2019-12-23 14:06:20.000000000,1970-01-01 00:00:00.000000001,open,0,"When the try statement includes return statement, all errors in the except statement are ignored."
481,2019-12-23 12:50:27.000000000,1970-01-01 00:00:00.000000001,open,4,merge-pyi doesn't work with pyi file with backticks generated by pytype
480,2019-12-21 03:35:55.000000000,2019-12-31 11:01:58.000000000,closed,3,Support alternate function argument comment syntax
479,2019-12-20 03:54:03.000000000,2019-12-20 04:29:39.000000000,closed,0,Google sync
478,2019-12-18 01:30:34.000000000,2019-12-18 02:35:41.000000000,closed,0,Google sync
477,2019-12-13 05:27:53.000000000,2019-12-13 06:10:36.000000000,closed,0,Fix broken links in pytype's PyPI description.
476,2019-12-12 05:48:32.000000000,2019-12-12 07:36:05.000000000,closed,0,Make pytype work with the most recent version of pylint.
475,2019-12-12 05:01:17.000000000,2019-12-12 05:39:27.000000000,closed,0,Remove an unnecessary list comprehension.
474,2019-12-11 02:24:03.000000000,2019-12-11 03:29:55.000000000,closed,0,Fixed AttributeError in Python 3.8 (#466) (#471)
473,2019-12-11 01:58:36.000000000,1970-01-01 00:00:00.000000001,open,0,pytype doesn't respect sys.version_info tuple checks
472,2019-12-11 01:31:13.000000000,2019-12-11 02:03:45.000000000,closed,0,Google sync
471,2019-12-08 14:00:09.000000000,2019-12-11 03:29:55.000000000,closed,6,Fixed AttributeError in Python 3.8 (#466)
470,2019-12-07 23:13:43.000000000,2019-12-11 05:05:40.000000000,closed,9,pyxref crash (KeyError with a dataclass field)
469,2019-12-07 16:27:31.000000000,2019-12-11 02:03:47.000000000,closed,1,missing-parameter on a derived dataclass
468,2019-12-07 00:42:25.000000000,2019-12-07 01:56:14.000000000,closed,0,Google sync
467,2019-12-06 07:50:29.000000000,2019-12-06 08:31:22.000000000,closed,0,Update typeshed to f7f68607ab910595d0076b2003d78a364f96c8a8.
466,2019-12-05 13:13:10.000000000,2019-12-11 03:29:54.000000000,closed,3,"""AttributeError: module 'time' has no attribute 'clock'"" when running pytype in Python 3.8"
465,2019-12-04 06:43:18.000000000,2019-12-04 07:40:06.000000000,closed,0,Google sync
464,2019-12-03 19:00:44.000000000,2020-01-15 19:23:36.000000000,closed,4,import errors with native namespace packages
463,2019-11-28 02:19:24.000000000,2019-11-30 00:25:19.000000000,closed,0,Remove pyi_checker from pytype's setup.py.
462,2019-11-27 23:47:56.000000000,2019-11-28 01:50:10.000000000,closed,0,Google sync
461,2019-11-27 05:25:09.000000000,2019-11-27 06:00:56.000000000,closed,0,Google sync
460,2019-11-26 22:10:23.000000000,1970-01-01 00:00:00.000000001,open,3,kwargs.update should allow arguments of different types
459,2019-11-24 18:12:29.000000000,1970-01-01 00:00:00.000000001,open,1,Enum functional APIÂ throw error
458,2019-11-24 16:39:21.000000000,2019-11-28 02:38:45.000000000,closed,3,python -m pytype.single
457,2019-11-23 21:57:18.000000000,2019-11-27 06:00:57.000000000,closed,2,Fix UnboundLocalError about 'argument_parser' in Options
456,2019-11-20 06:51:53.000000000,2019-11-20 10:33:34.000000000,closed,0,Google sync
455,2019-11-19 01:14:33.000000000,1970-01-01 00:00:00.000000001,open,6,Import error when using `from X import Y`
454,2019-11-18 19:07:05.000000000,2019-11-27 06:00:57.000000000,closed,1,Add a -j/--jobs option to pytype
453,2019-11-18 18:39:32.000000000,2019-11-20 10:33:35.000000000,closed,4,Add a top-level __main__.py module
452,2019-11-14 03:49:27.000000000,2019-11-14 04:25:06.000000000,closed,0,Google sync
451,2019-11-09 03:08:01.000000000,2019-11-09 04:02:27.000000000,closed,0,Google sync
450,2019-11-08 12:15:07.000000000,1970-01-01 00:00:00.000000001,open,2,A function reading a global variable and a loop causes an error.
449,2019-11-08 11:44:06.000000000,1970-01-01 00:00:00.000000001,open,1,Pytype seems do not check builtin map function
448,2019-11-07 21:49:06.000000000,1970-01-01 00:00:00.000000001,open,1,"When using ctypes, an Array field in a Structure causes an error."
447,2019-11-07 01:47:33.000000000,2019-11-07 02:23:24.000000000,closed,0,Google sync
446,2019-11-06 12:41:41.000000000,1970-01-01 00:00:00.000000001,open,1,overloading methods return with Generic type
445,2019-11-05 04:03:31.000000000,1970-01-01 00:00:00.000000001,open,3,Overloading return type with None
444,2019-11-01 20:27:40.000000000,2019-12-11 02:03:46.000000000,closed,1,Issue with specialization of aliases
443,2019-11-01 18:45:06.000000000,2020-02-01 07:22:44.000000000,closed,5,typing.Collection shouldn't inherit from Container.
442,2019-10-28 22:17:08.000000000,2019-10-28 23:01:46.000000000,closed,0,Google sync
441,2019-10-24 19:48:48.000000000,1970-01-01 00:00:00.000000001,open,5,[import-error] on Travis-CI
440,2019-10-23 13:47:43.000000000,1970-01-01 00:00:00.000000001,open,10,Support Python 3.8
439,2019-10-18 02:29:00.000000000,2019-10-18 03:02:09.000000000,closed,0,Google sync
438,2019-10-17 02:48:25.000000000,2019-10-17 03:36:26.000000000,closed,0,Documentation Edits wording/grammer/cleanup (#431)
437,2019-10-16 22:39:11.000000000,2019-10-18 03:02:10.000000000,closed,1,Python exe can't handle space in path
436,2019-10-16 03:51:18.000000000,2019-10-16 04:43:36.000000000,closed,0,Update README.md (#433)
435,2019-10-16 02:44:29.000000000,2019-10-16 03:16:43.000000000,closed,0,Google sync
434,2019-10-13 21:42:39.000000000,2019-10-16 03:16:44.000000000,closed,0,Problem with dangling comma
433,2019-10-13 07:01:40.000000000,2019-10-16 04:43:36.000000000,closed,1,Update README.md
432,2019-10-12 21:47:44.000000000,2019-10-16 03:16:44.000000000,closed,1,Empty tuple syntax
431,2019-10-12 10:55:24.000000000,2019-10-17 03:36:26.000000000,closed,3,Documentation Edits wording/grammer/cleanup
430,2019-10-10 23:22:49.000000000,2019-10-11 00:19:27.000000000,closed,0,Google sync
429,2019-10-08 02:30:45.000000000,2019-10-08 03:03:09.000000000,closed,0,Google sync
428,2019-10-02 06:54:29.000000000,2019-10-04 20:37:22.000000000,closed,3,Pyi error when running on larger codebases -- can't import for ''
427,2019-09-28 03:10:11.000000000,2019-09-28 03:45:56.000000000,closed,0,Copy the numpy stub to pytype so it can be used externally.
426,2019-09-28 01:18:44.000000000,2019-09-28 01:52:50.000000000,closed,0,Google sync
425,2019-09-28 00:09:27.000000000,2019-09-28 00:42:38.000000000,closed,0,Resolve some TODOs in pytype.
424,2019-09-27 22:49:14.000000000,2019-09-27 23:20:31.000000000,closed,0,Temporarily pin to an older version of pylint.
423,2019-09-27 22:10:14.000000000,2019-12-12 07:36:05.000000000,closed,1,"pytype not compatible with pylint 2.4.0+, astroid 2.3.0+"
422,2019-09-26 02:44:04.000000000,1970-01-01 00:00:00.000000001,open,1,Create and publish a pytest plugin
421,2019-09-23 23:11:05.000000000,2019-09-24 00:39:38.000000000,closed,0,Google sync
420,2019-09-23 12:53:42.000000000,2019-12-11 02:03:47.000000000,closed,2,3.7+: support `from __future__ import annotations`
419,2019-09-20 21:27:51.000000000,2019-11-09 04:02:29.000000000,closed,2,"""else"" causes error"
418,2019-09-19 04:19:07.000000000,2019-09-20 04:08:24.000000000,closed,0,Google sync
417,2019-09-18 12:28:52.000000000,1970-01-01 00:00:00.000000001,open,1,Recursive protocols type inference not working
416,2019-09-18 01:25:44.000000000,2019-09-18 02:01:36.000000000,closed,0,Google sync
415,2019-09-17 23:35:12.000000000,2019-09-18 02:01:49.000000000,closed,0,Add pytype/overlays/ as a new subpackage.
414,2019-09-16 19:40:30.000000000,1970-01-01 00:00:00.000000001,open,3,Can't find module 'passlib.hash'. [import-error]
413,2019-09-16 19:37:03.000000000,2019-09-18 10:16:08.000000000,closed,3,No attribute 'from_bytes' on Type[int] [attribute-error]
412,2019-09-15 23:24:10.000000000,2019-09-17 00:19:13.000000000,closed,2,Temporary fix for sorting errors
411,2019-09-12 20:06:50.000000000,1970-01-01 00:00:00.000000001,open,1,merge-pyi should handle overloaded definitions
410,2019-09-12 02:18:21.000000000,1970-01-01 00:00:00.000000001,open,1,AssertionError in match_Signature_against_Signature
409,2019-09-12 01:43:43.000000000,2020-01-08 04:05:02.000000000,closed,2,Caught error in pytype: 'str' object has no attribute 'get_instance_type'
408,2019-09-11 00:38:15.000000000,2019-12-18 02:35:42.000000000,closed,1,Does not handle @attr optional kw args
407,2019-09-11 00:06:35.000000000,1970-01-01 00:00:00.000000001,open,1,Don't crash on recursive type definitions
406,2019-09-08 16:39:41.000000000,2019-09-17 00:19:42.000000000,closed,5,Change default value of Error.filename to empty string instead of None
405,2019-09-08 12:17:33.000000000,1970-01-01 00:00:00.000000001,open,6,Cannot build on macos
404,2019-09-07 01:37:21.000000000,2019-09-07 04:01:31.000000000,closed,0,Update typeshed to 668988fa8cfd106a73218b23f953f226db89b3d4.
403,2019-09-06 23:44:53.000000000,2019-09-07 01:13:03.000000000,closed,0,Google sync
402,2019-09-05 23:32:47.000000000,2019-09-06 02:18:41.000000000,closed,0,Update typeshed to ce0f5b226fc00c732e5b6eb2a9fcb845cd18531c.
401,2019-09-02 12:43:43.000000000,2019-09-04 22:09:49.000000000,closed,2,unexpected List type inference behavior
400,2019-08-31 05:58:59.000000000,2019-09-04 22:02:54.000000000,closed,1,False error for @dataclass
399,2019-08-30 01:49:46.000000000,2019-08-30 02:32:44.000000000,closed,0,Update pytype __version__ and CHANGELOG for a PyPI release.
398,2019-08-29 04:15:07.000000000,2019-08-29 04:51:16.000000000,closed,0,Update typeshed to fab2ee0d7c38dba0a5142924e53460548fdce0b0.
397,2019-08-29 03:15:28.000000000,2019-08-29 03:51:43.000000000,closed,0,Google sync
396,2019-08-29 01:48:21.000000000,2019-08-29 02:39:26.000000000,closed,0,Add a verbose mode to print failing test logs to stderr.
395,2019-08-24 01:13:40.000000000,2019-08-24 05:20:28.000000000,closed,0,Google sync
394,2019-08-23 07:33:43.000000000,2019-08-23 21:12:52.000000000,closed,0,Don't lint or test xref in Python 2.
393,2019-08-22 01:44:27.000000000,2019-08-22 01:44:51.000000000,closed,1,error when building wheel in WSL
392,2019-08-22 01:37:40.000000000,2019-08-23 05:46:37.000000000,closed,4,add WSL directions
391,2019-08-21 20:15:55.000000000,2019-09-28 03:45:56.000000000,closed,2,isinstance() doesn't prevent type errors for some types
390,2019-08-16 03:40:08.000000000,2019-08-16 03:59:04.000000000,closed,0,Add attrs as a dependency
389,2019-08-15 17:44:17.000000000,2019-09-18 02:26:44.000000000,closed,4,sys.flags.utf8_mode not supported
388,2019-08-15 03:33:35.000000000,2019-08-15 05:49:57.000000000,closed,0,Google sync
387,2019-08-14 23:24:57.000000000,2019-08-15 00:07:43.000000000,closed,1,Update typeshed to 6f01493edc8a6f563e571121483679fd46db336a from August 7.
386,2019-08-14 02:19:35.000000000,1970-01-01 00:00:00.000000001,open,2,"""No attribute 'Pattern' on module 're'"" reported incorrectly"
385,2019-08-13 22:39:59.000000000,2019-08-14 10:04:57.000000000,closed,7,Question: usage as a library
384,2019-08-13 21:11:39.000000000,2019-09-18 10:45:31.000000000,closed,3,pytype crashed due to comparison between str and None
383,2019-08-13 20:26:53.000000000,1970-01-01 00:00:00.000000001,open,2,"When pytype finds an error in a method signature, it prints a confusing error message about calling the method even if it is only declared."
382,2019-08-10 04:02:21.000000000,2019-08-10 04:37:09.000000000,closed,0,Google sync
381,2019-08-10 02:23:44.000000000,2019-08-10 02:56:39.000000000,closed,2,Use README.md for the PyPI description.
380,2019-08-09 23:10:25.000000000,2019-08-09 23:46:15.000000000,closed,0,Add an entrypoint for annotate-ast to setup.py.
379,2019-08-09 15:12:14.000000000,1970-01-01 00:00:00.000000001,open,3,TypeVar may appear only once in signature if used in the body
378,2019-08-09 03:29:59.000000000,2019-08-09 23:02:21.000000000,closed,0,Google sync
377,2019-08-09 02:52:44.000000000,2019-08-09 03:29:02.000000000,closed,0,Install typed_ast before running Travis tests.
376,2019-08-08 18:00:40.000000000,1970-01-01 00:00:00.000000001,open,1,Run tests against installed version of Python
375,2019-08-08 00:27:28.000000000,1970-01-01 00:00:00.000000001,open,3,Decorator on method returns wrong-arg-count
374,2019-08-07 10:15:15.000000000,2019-08-30 02:56:35.000000000,closed,8,"pytype does not recognize token.NL, token.COMMENT and token.ENCODING"
373,2019-08-06 02:40:39.000000000,2019-08-08 01:01:42.000000000,closed,1,Add new pytype/tools/ subdirs to `packages` in setup.py.
372,2019-08-06 01:11:05.000000000,2019-08-08 01:01:41.000000000,closed,3,pytype setup omits pytype.tools.traces
371,2019-08-04 02:17:10.000000000,2019-12-07 02:42:00.000000000,closed,4,posix.listdir is not found
370,2019-08-03 05:06:43.000000000,2019-08-03 05:51:54.000000000,closed,0,Google sync
369,2019-07-30 23:08:14.000000000,2019-07-30 23:40:46.000000000,closed,0,Allow trailing commas in lists in type stubs.
368,2019-07-30 04:39:27.000000000,2019-07-30 05:13:04.000000000,closed,0,Google sync
367,2019-07-26 04:50:18.000000000,2019-08-15 05:49:57.000000000,closed,1,__suppress_context__ not recognized
366,2019-07-24 19:28:27.000000000,2019-07-24 20:50:53.000000000,closed,0,Google sync
365,2019-07-23 05:03:05.000000000,1970-01-01 00:00:00.000000001,open,1,Not checking all possible parameter types for function calls
364,2019-07-23 01:54:31.000000000,2019-07-23 02:36:11.000000000,closed,0,Update typeshed to 40215d1fa3ef33a31fe68624472c10dd62350f18.
363,2019-07-22 15:18:14.000000000,1970-01-01 00:00:00.000000001,open,1,Add junit xml output format
362,2019-07-18 06:14:24.000000000,2020-04-28 02:00:34.000000000,closed,2,"Comment syntax for varargs, kwargs incompatible with PEP-484, mypy"
361,2019-07-17 10:26:03.000000000,2019-07-17 10:57:32.000000000,closed,0,Google sync
360,2019-07-17 04:20:47.000000000,2019-07-17 10:59:38.000000000,closed,3,docs: fix typo
359,2019-07-16 22:47:20.000000000,2019-07-17 00:12:20.000000000,closed,0,Google sync
358,2019-07-16 19:33:27.000000000,2019-08-21 07:27:39.000000000,closed,4,Updating Setup.cfg
357,2019-07-11 21:55:23.000000000,2019-07-11 22:34:09.000000000,closed,0,Google sync
356,2019-07-10 22:45:22.000000000,1970-01-01 00:00:00.000000001,open,1,io.TextIOWrapper.__init__() type error
355,2019-07-09 10:00:54.000000000,1970-01-01 00:00:00.000000001,open,2,pyi-error couldn't import pyi for 'MyClassName'
354,2019-07-04 01:56:51.000000000,2019-07-04 03:24:32.000000000,closed,0,Google sync
353,2019-07-02 22:31:08.000000000,2019-07-03 01:40:21.000000000,closed,0,Google sync
352,2019-06-28 03:19:44.000000000,2019-06-28 03:52:30.000000000,closed,0,Google sync
351,2019-06-27 04:58:10.000000000,2019-06-27 05:30:23.000000000,closed,0,Google sync
350,2019-06-22 16:54:16.000000000,1970-01-01 00:00:00.000000001,open,4,asyncio.transports.Transport attribute-error set_write_buffer_limits
349,2019-06-22 16:36:33.000000000,2019-06-28 03:52:31.000000000,closed,1,wrong-arg-types for UnicodeDecodeError exception
348,2019-06-22 13:23:15.000000000,2019-06-29 05:07:40.000000000,closed,3,pytype unable to find some codecs attributes
347,2019-06-20 22:19:12.000000000,1970-01-01 00:00:00.000000001,open,0,pyxref --debug doesn't output Kythe facts
346,2019-06-20 06:15:09.000000000,2019-06-20 06:51:11.000000000,closed,0,Google sync
345,2019-06-20 03:54:07.000000000,2019-06-20 04:31:44.000000000,closed,1,Google sync
344,2019-06-18 05:09:17.000000000,2019-06-20 11:07:03.000000000,closed,1,--noquick option?
343,2019-06-18 03:08:20.000000000,2019-06-18 04:49:47.000000000,closed,2,Update README.md
342,2019-06-15 05:16:15.000000000,2019-06-15 05:47:09.000000000,closed,0,Google sync
341,2019-06-08 10:34:41.000000000,2019-06-08 11:04:54.000000000,closed,0,Google sync
340,2019-06-08 08:49:57.000000000,2019-06-08 10:11:02.000000000,closed,1,Google sync
339,2019-06-07 03:15:07.000000000,2019-06-08 01:35:02.000000000,closed,2,Finish adding merge-pyi's local copy of the lib2to3 grammar.
338,2019-06-07 02:33:53.000000000,2019-06-07 03:07:35.000000000,closed,0,Google sync
337,2019-06-06 22:30:54.000000000,2019-06-07 02:04:33.000000000,closed,2,Pytype fails with caught ImportError
336,2019-06-05 22:19:36.000000000,2019-06-05 22:55:07.000000000,closed,0,Google sync
335,2019-06-05 17:35:42.000000000,1970-01-01 00:00:00.000000001,open,3,Run pytype with a default config [Enhancement]
334,2019-06-05 13:17:22.000000000,2019-06-05 21:03:56.000000000,closed,0,Support the load_tests protocol in build_scripts.test_module.
333,2019-06-05 06:57:27.000000000,2019-06-05 22:55:08.000000000,closed,0,merge-pyi prints too much logging output
332,2019-06-05 06:52:36.000000000,2019-06-08 01:35:02.000000000,closed,0,merge-pyi chokes on variable annotation syntax in Python 2
331,2019-06-05 03:31:20.000000000,2019-06-05 04:06:15.000000000,closed,0,Google sync
330,2019-06-03 20:09:55.000000000,2019-06-08 11:04:54.000000000,closed,1,merge-pyi imports 1 type per line
329,2019-06-01 00:53:47.000000000,2019-06-01 01:33:07.000000000,closed,0,Google sync
328,2019-06-01 00:10:03.000000000,2019-06-01 00:30:38.000000000,closed,1,Google sync
327,2019-05-30 23:52:58.000000000,2019-05-31 00:25:32.000000000,closed,0,Google sync
326,2019-05-30 00:08:42.000000000,1970-01-01 00:00:00.000000001,open,3,IDE integration
325,2019-05-24 23:07:38.000000000,2019-05-24 23:38:21.000000000,closed,0,Google sync
324,2019-05-24 22:51:15.000000000,2019-05-24 23:26:16.000000000,closed,0,Bump the required importlab version.
323,2019-05-24 11:28:34.000000000,1970-01-01 00:00:00.000000001,open,1,Variable type annotations are being ignored
322,2019-05-23 01:27:04.000000000,2019-07-04 03:24:33.000000000,closed,3,using functools.partial as a class leads to spurious errors
321,2019-05-21 22:04:13.000000000,1970-01-01 00:00:00.000000001,open,2,pytype doesn't support `x = y = z` in pyi files (seen in attr stubs)
320,2019-05-19 18:40:45.000000000,1970-01-01 00:00:00.000000001,open,1,Pytype error that there is no pyi file for a subpackage
319,2019-05-18 20:49:43.000000000,2019-05-20 21:05:02.000000000,closed,2,attribute-error for importlib.abc.Loader.exec_module
318,2019-05-18 20:40:06.000000000,2019-05-24 22:29:34.000000000,closed,3,import-error for jsonschema
317,2019-05-16 03:19:22.000000000,1970-01-01 00:00:00.000000001,open,0,pytype doesn't support ignoring a missing import with `# type: ignore` in pyi files (seen in fb303.FacebookService and scribe.scribe stubs)
316,2019-05-16 03:16:59.000000000,1970-01-01 00:00:00.000000001,open,0,pytype doesn't support aliasing an import from a class in pyi files (seen in pynamodb.models stub)
315,2019-05-16 03:11:12.000000000,1970-01-01 00:00:00.000000001,open,0,pytype doesn't support `from . import SomeClass` in pyi files (seen in attr stubs)
314,2019-05-15 21:37:08.000000000,2019-05-16 00:16:24.000000000,closed,0,Google sync
313,2019-05-15 06:21:02.000000000,2019-05-15 06:53:09.000000000,closed,0,Update typeshed to 50d98acc766e9425cb099c17b711f1bccb697584.
312,2019-05-15 03:23:10.000000000,2019-05-15 04:03:52.000000000,closed,0,Google sync
311,2019-05-14 18:49:20.000000000,2019-06-07 02:10:51.000000000,closed,4,Add setuptools integration
310,2019-05-12 21:48:29.000000000,2019-05-12 22:18:52.000000000,closed,0,Google sync
309,2019-05-09 05:36:24.000000000,2019-05-09 06:06:35.000000000,closed,0,Fix a bug where pytype would fail to resolve an unused import.
308,2019-05-09 04:25:36.000000000,2019-05-09 04:56:33.000000000,closed,0,Google sync
307,2019-05-08 21:39:26.000000000,2019-05-09 04:56:42.000000000,closed,4,Add missing comma
306,2019-05-07 01:58:32.000000000,2019-05-07 02:29:33.000000000,closed,0,Google sync
305,2019-05-05 14:57:14.000000000,2019-06-27 05:01:50.000000000,closed,5,"2019.04.26 installation fails with ""parser_ext.o: file not recognized"""
304,2019-05-03 12:04:43.000000000,1970-01-01 00:00:00.000000001,open,1,Syntax parser issues - import
303,2019-05-02 01:29:02.000000000,2019-05-02 02:27:52.000000000,closed,0,Update typeshed to 4e572ae6a398c82fa9f1b0e7ba7c819274de8c3a.
302,2019-04-26 22:22:56.000000000,2019-04-26 22:55:25.000000000,closed,0,Google sync
301,2019-04-24 22:51:40.000000000,2019-04-24 23:52:11.000000000,closed,0,Google sync
300,2019-04-24 13:04:58.000000000,2019-04-26 11:20:29.000000000,closed,5,pytype seems to ignore logging.setLoggerClass()
299,2019-04-22 16:54:19.000000000,2019-06-02 22:45:45.000000000,closed,9,WIP FIX:PATH issues when running in windows #90
298,2019-04-20 00:16:49.000000000,2019-04-20 00:49:07.000000000,closed,0,Google sync
297,2019-04-18 00:29:20.000000000,2019-04-18 01:40:55.000000000,closed,0,Google sync
296,2019-04-16 05:15:54.000000000,2019-04-18 01:40:56.000000000,closed,3,Invalid base class on Protocol (typing_extensions)
295,2019-04-12 23:50:24.000000000,2019-04-16 01:11:35.000000000,closed,1,Re-enable some disabled pylint messages.
294,2019-04-12 22:37:43.000000000,2019-04-12 23:14:33.000000000,closed,0,Google sync
293,2019-04-12 00:27:17.000000000,2019-04-12 02:55:25.000000000,closed,0,Run pylint over build_scripts/ and setup.py as well.
292,2019-04-11 01:36:15.000000000,2019-04-11 02:05:36.000000000,closed,0,Google sync
291,2019-04-10 04:12:19.000000000,2019-04-10 04:45:00.000000000,closed,0,Google sync
290,2019-04-10 02:32:47.000000000,2019-04-10 04:07:12.000000000,closed,0,Update typeshed pin to cfa65b831b327c595129fcc11c855ac504920d98.
289,2019-04-05 22:20:14.000000000,2019-04-05 22:56:46.000000000,closed,0,Bump pytype's PyPI version number for a release.
288,2019-04-04 22:29:49.000000000,2019-04-04 23:01:10.000000000,closed,0,Google sync
287,2019-04-04 19:55:24.000000000,2019-04-11 22:43:22.000000000,closed,3,Add pylintrc #278
286,2019-04-04 17:30:31.000000000,2019-04-04 23:01:10.000000000,closed,2,Match and __getitem__ error?
285,2019-04-04 01:23:50.000000000,1970-01-01 00:00:00.000000001,open,3,pytype swallows crash and outputs to .pyi
284,2019-04-03 05:33:20.000000000,2019-04-03 06:47:52.000000000,closed,0,Only require typed_ast in 3.3+.
283,2019-04-03 02:13:28.000000000,2019-04-03 02:46:24.000000000,closed,0,Change our development status from Stable to Production/Stable.
282,2019-04-03 01:31:38.000000000,2019-04-03 02:03:44.000000000,closed,0,Google sync
281,2019-04-02 02:07:09.000000000,2019-04-02 02:42:54.000000000,closed,0,Google sync
280,2019-04-02 00:02:17.000000000,2019-04-02 00:41:00.000000000,closed,0,Add a dependency on typed_ast
279,2019-04-01 23:14:23.000000000,2019-04-02 00:03:57.000000000,closed,0,Update typeshed to 9b9ff64fc59f95e3595cfdfd821675d81ad56aa3.
278,2019-03-30 05:33:00.000000000,2019-04-11 23:19:58.000000000,closed,7,Use pylint
277,2019-03-30 04:51:56.000000000,2019-04-02 02:42:55.000000000,closed,2,Shorten intermediate output.
276,2019-03-30 02:30:22.000000000,2019-03-30 03:07:55.000000000,closed,0,Google sync
275,2019-03-30 01:36:01.000000000,2019-03-30 02:30:54.000000000,closed,3,add an executable for xref to setup.py
274,2019-03-29 21:55:44.000000000,2019-05-07 02:29:34.000000000,closed,1,pytype fails analyzing module that imports requests
273,2019-03-27 22:59:20.000000000,2019-03-28 01:17:53.000000000,closed,0,Google sync
272,2019-03-27 05:39:59.000000000,2019-03-28 01:17:54.000000000,closed,7,made `python_verision` in pytype.cfg same as its runtime python version
271,2019-03-26 15:50:03.000000000,2019-03-26 21:16:34.000000000,closed,1,Travis CI: The 'sudo' tag is now deprecated
270,2019-03-26 08:37:10.000000000,2019-03-28 01:38:37.000000000,closed,3,`python_version` in generated pytype.cfg is different from its python version
269,2019-03-23 01:59:37.000000000,2019-10-12 00:18:21.000000000,closed,22,xref crashes with AttributeError: 'LateAnnotation' object has no attribute 'Bindings'
268,2019-03-22 01:22:04.000000000,2019-03-22 01:54:03.000000000,closed,0,Google sync
267,2019-03-22 00:03:00.000000000,2019-03-22 01:07:23.000000000,closed,0,Update the typeshed pin to match the internal version.
266,2019-03-21 22:56:37.000000000,2019-03-21 23:23:01.000000000,closed,1,attribute-error with metaclass
265,2019-03-21 20:17:55.000000000,2019-03-26 21:28:59.000000000,closed,5,Default ast library
264,2019-03-19 09:03:22.000000000,1970-01-01 00:00:00.000000001,open,3,pytype gets the wrong import when __init__ collides with file
263,2019-03-19 00:21:08.000000000,2019-03-19 00:52:24.000000000,closed,0,Google sync
262,2019-03-16 23:58:33.000000000,2019-03-17 00:27:08.000000000,closed,1,Google sync
261,2019-03-16 00:10:13.000000000,2019-03-16 08:02:15.000000000,closed,1,Google sync
260,2019-03-14 22:26:41.000000000,2019-03-16 00:00:06.000000000,closed,1,Remove outdated reference to asyncio from setup.py
259,2019-03-14 22:26:22.000000000,2019-03-15 22:55:58.000000000,closed,1,Google sync
258,2019-03-12 22:39:13.000000000,2019-03-13 05:13:39.000000000,closed,0,Google sync
257,2019-03-10 04:29:24.000000000,2019-03-13 05:13:40.000000000,closed,0,Re-export renamed typing module members.
256,2019-03-09 06:27:59.000000000,2019-03-13 05:16:46.000000000,closed,4,Can't find module 'contextlib.AbstractContextManager'
255,2019-03-09 06:24:59.000000000,1970-01-01 00:00:00.000000001,open,4,Spurious attribute-error for '__init_subclass__'
254,2019-03-09 00:49:23.000000000,2019-03-09 01:16:14.000000000,closed,0,Google sync
253,2019-03-08 00:38:49.000000000,2019-03-09 00:06:11.000000000,closed,1,Google sync
252,2019-03-07 04:25:02.000000000,2019-03-07 06:39:49.000000000,closed,0,Google sync
251,2019-03-02 05:32:42.000000000,2019-03-02 06:06:33.000000000,closed,0,Google sync
250,2019-03-02 00:12:25.000000000,2019-03-02 04:24:40.000000000,closed,1,Google sync
249,2019-03-01 09:59:45.000000000,2019-03-02 06:06:34.000000000,closed,0,Add test_data to MANIFEST.in
248,2019-02-28 06:59:12.000000000,2019-03-02 04:24:41.000000000,closed,1,Include LICENSE in MANIFEST.in.
247,2019-02-28 01:18:15.000000000,2019-02-28 01:30:10.000000000,closed,0,Google sync
246,2019-02-27 15:10:16.000000000,2019-03-02 04:40:39.000000000,closed,2,Add license to sdist
245,2019-02-27 14:41:26.000000000,2019-03-06 23:18:30.000000000,closed,3,Add test data to sdist
244,2019-02-19 07:33:55.000000000,2019-06-22 04:16:22.000000000,closed,6,"Awaitable vs. Generator[Any, None, ResultType] in asyncio API's [bad-return-type]"
243,2019-02-19 07:09:02.000000000,2019-03-19 00:52:25.000000000,closed,3,bytearray.clear() and bytearray.copy() report as attribute-error
242,2019-02-16 05:22:18.000000000,2019-04-02 02:42:55.000000000,closed,1,"Missing typing definitions: ChainMap, Counter"
241,2019-02-15 06:12:30.000000000,2019-03-02 04:24:42.000000000,closed,0,Pytype can't analyze Python 3.4
240,2019-02-14 05:11:18.000000000,2019-02-14 06:23:06.000000000,closed,0,Google sync
239,2019-02-12 23:51:12.000000000,2019-12-13 06:10:36.000000000,closed,1,PyPI project page includes broken links
238,2019-02-12 06:00:40.000000000,2019-02-12 06:33:04.000000000,closed,0,Update the PyPI url in setup.py
237,2019-02-09 05:16:55.000000000,2019-02-09 05:54:04.000000000,closed,0,Google sync
236,2019-02-09 00:52:33.000000000,2019-02-09 02:30:36.000000000,closed,0,Google sync
235,2019-01-31 00:24:04.000000000,2019-01-31 02:01:31.000000000,closed,1,Google sync
234,2019-01-29 10:49:16.000000000,2019-12-02 15:56:10.000000000,closed,11,Support for dataclasses
233,2019-01-26 03:12:55.000000000,2019-01-29 04:32:06.000000000,closed,2,Google sync
232,2019-01-21 20:55:43.000000000,2019-05-16 03:07:17.000000000,closed,2,Error importing pyi for jinja2.
231,2019-01-19 03:52:35.000000000,2019-01-19 05:44:35.000000000,closed,3,Google sync
230,2019-01-18 04:33:19.000000000,2019-01-18 05:48:13.000000000,closed,0,Google sync
229,2019-01-17 20:20:35.000000000,2019-01-18 05:48:14.000000000,closed,6,#228 Fix parsing of Python version string.
228,2019-01-17 20:18:12.000000000,2019-01-24 00:46:00.000000000,closed,1,Pytype does not work with Python distributions like Anaconda
227,2019-01-15 23:21:39.000000000,2019-02-14 06:23:07.000000000,closed,0,Improve handling of pyi circular dependencies
226,2019-01-15 19:49:16.000000000,2019-03-08 06:53:05.000000000,closed,2,Over approximation 
225,2019-01-09 03:48:13.000000000,2019-03-19 00:52:25.000000000,closed,0,BaseException is missing attributes
224,2019-01-09 03:10:06.000000000,2019-01-09 04:51:27.000000000,closed,0,Google sync
223,2019-01-08 04:38:50.000000000,2019-01-09 02:03:14.000000000,closed,1,Google sync
222,2019-01-05 07:21:59.000000000,2019-01-05 11:09:21.000000000,closed,0,Google sync
221,2018-12-23 20:55:02.000000000,2019-03-26 15:46:45.000000000,closed,2,Travis CI: Use flake8 to find syntax errors and undefined names
220,2018-12-22 05:58:48.000000000,2018-12-26 23:14:13.000000000,closed,0,Google sync
219,2018-12-21 03:02:25.000000000,2019-03-16 08:02:16.000000000,closed,3,False error on keywords only arguments
218,2018-12-20 04:08:41.000000000,2018-12-20 04:42:46.000000000,closed,0,Google sync
217,2018-12-19 14:09:59.000000000,2018-12-20 04:42:46.000000000,closed,1,Add testing framework for target Python 3.7.
216,2018-12-19 06:56:54.000000000,2019-02-09 02:30:37.000000000,closed,2,Variable annotations are broken in Python 3.7
215,2018-12-19 03:54:23.000000000,1970-01-01 00:00:00.000000001,open,6,Inferred TypeVar information lost through function calls
214,2018-12-19 02:38:44.000000000,2019-04-18 01:40:55.000000000,closed,1,Protocols require ABCMeta to work
213,2018-12-19 02:05:02.000000000,2018-12-19 02:27:02.000000000,closed,0,"Remove "".py"" report (#204)"
212,2018-12-19 00:29:16.000000000,2018-12-26 23:14:14.000000000,closed,0,Pytype should show detailed error information for an unparseable file and analyze the remaining inputs
211,2018-12-18 06:08:13.000000000,1970-01-01 00:00:00.000000001,open,0,Pytype fails to discover namedtuple constructor in child class
210,2018-12-18 03:42:00.000000000,2018-12-18 04:35:14.000000000,closed,0,Google sync
209,2018-12-18 00:49:06.000000000,2020-04-22 05:31:50.000000000,closed,4,Receive invalid-typevar error when using `type(arg)` with a typevar'd input
208,2018-12-17 00:24:29.000000000,2018-12-20 04:42:47.000000000,closed,1,"python 3.7 travis tests are using target python_version = (3, 6)"
207,2018-12-15 21:49:57.000000000,2018-12-18 19:58:42.000000000,closed,1,"Remove tests for "".py"""
206,2018-12-15 02:47:50.000000000,2018-12-15 05:47:28.000000000,closed,1,How to solve  build.ninja error
205,2018-12-14 21:15:22.000000000,2018-12-18 04:35:15.000000000,closed,10,Add unreadable_python_inputs
204,2018-12-14 01:23:11.000000000,2018-12-19 02:27:02.000000000,closed,11,"Remove "".py"" report"
203,2018-12-14 00:57:03.000000000,2019-01-29 04:32:07.000000000,closed,2,"""import asyncio"" failure in python 3.7"
202,2018-12-13 04:16:31.000000000,2019-04-02 02:42:55.000000000,closed,0,Add an option to not drop pytype_output in the current directory.
201,2018-12-12 05:59:33.000000000,2018-12-13 03:19:31.000000000,closed,2,Google sync
200,2018-12-11 01:09:05.000000000,1970-01-01 00:00:00.000000001,open,1,pytype should not treat `None` as a valid boolean value
199,2018-12-05 07:28:09.000000000,2018-12-06 03:11:16.000000000,closed,0,Use https as the website url scheme
198,2018-12-04 00:58:23.000000000,1970-01-01 00:00:00.000000001,open,7,Possible name collision for same-named source file in different directories
197,2018-12-04 00:25:40.000000000,2020-04-29 00:52:38.000000000,closed,1,Generic classes should have a register method from ABCMeta
196,2018-11-20 01:04:24.000000000,2018-11-20 02:12:12.000000000,closed,0,Google sync
195,2018-11-16 06:19:02.000000000,1970-01-01 00:00:00.000000001,open,3,merge-pyi: add an overwrite option
194,2018-11-10 22:21:11.000000000,2018-11-20 02:12:13.000000000,closed,3,[pytype 2018.11.06] CRITICAL Cannot parse input files:
193,2018-11-07 05:51:49.000000000,2019-11-20 10:34:26.000000000,closed,1,"pytype should be runnable with ""python -m pytype"""
192,2018-11-07 05:49:47.000000000,2018-12-13 03:19:32.000000000,closed,3,pytype should use the environment that it is run in by default
191,2018-11-07 03:51:22.000000000,2018-11-07 04:28:05.000000000,closed,0,Google sync
190,2018-11-07 01:42:35.000000000,2018-11-07 02:11:42.000000000,closed,1,Google sync
189,2018-11-02 01:53:22.000000000,2018-11-02 02:51:08.000000000,closed,1,Have pytype (naively) generate ninja files.
188,2018-11-01 05:18:59.000000000,2018-11-01 05:26:35.000000000,closed,0,Remove custom email notification settings from .travis.yml.
187,2018-10-31 01:24:20.000000000,2018-10-31 02:37:37.000000000,closed,0,Google sync
186,2018-10-28 08:44:11.000000000,2018-10-29 05:32:46.000000000,closed,0,Update typeshed to 9ee458226c212acc2691f014d2a5575f33160d6d.
185,2018-10-25 00:25:12.000000000,2018-11-01 05:26:35.000000000,closed,0,create a separate mailing list for travis notifications
184,2018-10-17 22:18:47.000000000,2018-10-17 22:51:05.000000000,closed,0,Google sync
183,2018-10-17 03:33:39.000000000,2018-10-17 03:36:21.000000000,closed,1,Google sync
182,2018-10-16 04:17:21.000000000,2018-10-16 04:51:29.000000000,closed,0,Update typeshed to cec2202550ab0f41205fad521a889415be0fd405.
181,2018-10-09 02:25:50.000000000,2018-10-13 10:50:07.000000000,closed,2,high memory usage
180,2018-10-06 04:03:03.000000000,2018-10-06 05:04:40.000000000,closed,0,Turn on type-checking for more of pytype itself.
179,2018-10-06 01:21:18.000000000,2018-10-06 03:56:10.000000000,closed,0,Google sync
178,2018-10-05 07:54:55.000000000,2018-10-06 03:56:11.000000000,closed,1,bytearray.__delitem__ should support a slice argument?
177,2018-10-05 00:34:52.000000000,2018-10-05 01:37:41.000000000,closed,1,Google sync
176,2018-10-04 02:06:26.000000000,2018-10-04 02:34:07.000000000,closed,2,Google sync
175,2018-09-27 05:11:34.000000000,2018-10-05 01:37:42.000000000,closed,1,Pytype sometimes misses import-related module-attr errors
174,2018-09-27 05:09:14.000000000,2019-03-16 08:02:16.000000000,closed,0,verify_ast() isn't called by --parse-pyi
173,2018-09-27 04:55:01.000000000,2018-11-07 04:28:06.000000000,closed,2,have pytype generate ninja files
172,2018-09-27 04:51:29.000000000,1970-01-01 00:00:00.000000001,open,0,Allow specifying multiple target python versions
171,2018-09-27 04:47:06.000000000,2019-05-16 03:20:41.000000000,closed,1,typeshed/pytype_test.py doesn't cover all of typeshed/third_party/
170,2018-09-27 04:43:45.000000000,1970-01-01 00:00:00.000000001,open,0,Have pytype-all issue a warning upon encountering a bad option in a config file
169,2018-09-27 04:42:31.000000000,2018-11-02 02:51:07.000000000,closed,0,pytype-all should stop analyzing a cycle if all source files have been checked.
168,2018-09-27 04:07:22.000000000,2018-11-07 04:28:06.000000000,closed,0,Add tags and changelogs for major releases
167,2018-09-27 04:02:48.000000000,1970-01-01 00:00:00.000000001,open,0,Enable use of pytype to generate pyi files for typeshed
166,2018-09-26 21:06:06.000000000,2018-09-26 23:54:12.000000000,closed,0,Google sync
165,2018-09-26 09:45:31.000000000,2018-09-26 20:10:26.000000000,closed,0,Update the typeshed submodule.
164,2018-09-26 00:18:29.000000000,2018-09-26 01:22:50.000000000,closed,0,Google sync
163,2018-09-25 23:38:32.000000000,1970-01-01 00:00:00.000000001,open,1,Support try/except ImportError
162,2018-09-21 04:18:50.000000000,2018-09-25 11:14:35.000000000,closed,2,Move pytype options out of travis_script into setup.cfg.
161,2018-09-20 22:57:49.000000000,2018-09-20 23:59:36.000000000,closed,0,Update the typeshed submodule.
160,2018-09-20 21:52:03.000000000,2018-09-20 22:17:31.000000000,closed,0,Google sync
159,2018-09-20 12:59:30.000000000,2018-10-05 01:37:42.000000000,closed,4,ParseError: Duplicate top-level identifier(s): _N
158,2018-09-19 00:45:47.000000000,2018-09-19 02:21:19.000000000,closed,1,Google sync
157,2018-09-18 22:46:31.000000000,2018-09-19 00:39:21.000000000,closed,1,Google sync
156,2018-09-13 21:29:36.000000000,2018-09-14 02:07:38.000000000,closed,0,Organize Travis commands into steps.
155,2018-09-13 10:51:44.000000000,2018-09-13 19:46:26.000000000,closed,0,Google sync
154,2018-09-12 01:41:07.000000000,2018-09-13 02:42:54.000000000,closed,1,Nondeterministic error messages
153,2018-09-12 01:18:58.000000000,2018-09-12 01:51:59.000000000,closed,0,Add Python3.7 host testing to Travis matrix.
152,2018-09-10 22:11:24.000000000,2019-01-05 11:09:21.000000000,closed,0,Open-source the pytype FAQ
151,2018-09-10 20:48:22.000000000,1970-01-01 00:00:00.000000001,open,2,Support PEP 561 (Distributing and Packaging Type Information)
150,2018-09-08 04:28:44.000000000,2019-05-15 06:53:09.000000000,closed,1,Pytype cannot handle <class name>.<nested class name> in pyi files
149,2018-09-08 03:27:26.000000000,2018-09-08 03:54:43.000000000,closed,0,Update the typeshed submodule.
148,2018-09-08 02:29:34.000000000,2018-09-08 02:47:55.000000000,closed,0,Google sync
147,2018-09-08 01:43:25.000000000,2018-09-08 02:47:56.000000000,closed,0,Add partial support for nested classes in pyi files.
146,2018-09-08 00:35:36.000000000,2018-09-08 00:58:26.000000000,closed,0,Google sync
145,2018-09-07 09:27:41.000000000,2018-09-07 10:06:09.000000000,closed,0,Add back the cpython submodule.
144,2018-09-05 01:27:00.000000000,2018-09-05 02:03:18.000000000,closed,0,Future-proof preconditions.py:_TOKEN_RE
143,2018-09-05 00:59:11.000000000,2018-09-05 01:05:16.000000000,closed,0,Refactor: Move the kythe graph generation into its own file.
142,2018-09-05 00:10:10.000000000,2018-09-05 02:03:18.000000000,closed,1,Future-proof preconditions.py:_TOKEN_RE
141,2018-09-04 22:53:22.000000000,2018-09-05 00:29:02.000000000,closed,0,Add cpython submodule pinned to the current 2.7 HEAD.
140,2018-08-31 21:12:56.000000000,2018-09-05 02:03:56.000000000,closed,1,FutureWarning under Py3.7: Possible nested set (preconditions.py:166)
139,2018-08-29 21:36:12.000000000,2018-08-29 21:54:18.000000000,closed,3,Google sync
138,2018-08-24 22:52:55.000000000,2018-08-24 23:15:20.000000000,closed,0,Adjust run_tests.py to reflect the fact that it also runs C++ unit tests.
137,2018-08-24 02:42:35.000000000,2018-08-24 03:01:38.000000000,closed,0,Google sync
136,2018-08-21 02:25:43.000000000,2018-08-23 20:51:55.000000000,closed,1,Add a simple script to run a C++ unit test binary.
135,2018-08-21 01:11:56.000000000,2018-08-21 01:41:07.000000000,closed,0,Google sync
134,2018-08-19 02:44:37.000000000,2018-09-08 04:34:47.000000000,closed,3,pytype chokes on Python 3 yaml (upstream typeshed issue?)
133,2018-08-18 04:10:46.000000000,2018-09-08 04:11:27.000000000,closed,14,pytype chokes on python 3 testtools
132,2018-08-18 01:31:24.000000000,2018-08-18 01:50:05.000000000,closed,0,Add back the googletest submodule.
131,2018-08-18 00:31:01.000000000,2018-08-18 00:55:07.000000000,closed,0,Google sync
130,2018-08-15 22:38:20.000000000,2018-08-16 10:54:34.000000000,closed,1,Add googletest as a submodule.
129,2018-08-15 01:10:52.000000000,2018-08-15 03:16:58.000000000,closed,0,Google sync
128,2018-08-11 09:27:43.000000000,2018-09-25 04:34:45.000000000,closed,1,"networkx import clobbers unittest, confusing six.pyi"
127,2018-08-10 21:41:43.000000000,2019-03-19 00:52:25.000000000,closed,0,sys.version_info should be a 5-tuple with extra attributes
126,2018-08-10 15:05:23.000000000,2018-08-10 20:22:36.000000000,closed,0,Dogfood open-source pytype.
125,2018-08-10 12:24:42.000000000,2018-08-10 12:32:26.000000000,closed,0,Internal copy of https://github.com/google/pytype/pull/124.
124,2018-08-10 07:23:08.000000000,2018-08-10 11:02:22.000000000,closed,2,Support excluding files or directories from analyze_project.
123,2018-08-10 03:00:07.000000000,2018-08-10 03:21:31.000000000,closed,1,Google sync
122,2018-08-10 01:23:35.000000000,2018-08-10 01:40:23.000000000,closed,0,Allow pytype to override an existing logging level. (#117)
121,2018-08-10 00:56:25.000000000,2018-08-10 10:30:33.000000000,closed,4,Update the type annotations backporting patch.
120,2018-08-10 00:33:03.000000000,2018-08-10 03:21:32.000000000,closed,3,Fix compiler warnings due to assigning const char* values to char* variables.
119,2018-08-09 22:57:48.000000000,2018-08-10 01:12:17.000000000,closed,0,Google sync
118,2018-08-09 20:41:46.000000000,2018-08-09 22:52:13.000000000,closed,3,Google sync
117,2018-08-09 09:28:51.000000000,2018-08-10 01:40:23.000000000,closed,3,Allow pytype to override an existing logging level.
116,2018-08-07 20:49:14.000000000,2018-08-07 23:07:04.000000000,closed,0,"Fix repeated ""the"" in CONTRIBUTING.md"
115,2018-08-06 14:52:33.000000000,2018-08-07 00:26:36.000000000,closed,0,Google sync
114,2018-08-06 12:53:31.000000000,2018-08-07 20:45:48.000000000,closed,1,Add --debug option to the scripts build.py and run_tests.py
113,2018-08-04 02:28:15.000000000,2018-08-06 14:44:25.000000000,closed,7,Google sync
112,2018-08-04 01:43:57.000000000,2018-08-07 00:26:38.000000000,closed,0,Fix a typo in parser.py.
111,2018-08-03 16:56:07.000000000,2020-06-23 09:12:36.000000000,closed,11,Don't require .py extension
110,2018-08-02 22:54:23.000000000,2018-08-03 06:11:12.000000000,closed,1,Google sync
109,2018-08-02 22:45:55.000000000,2018-08-04 10:22:35.000000000,closed,0,Add a convenience script to build all pytype targets.
108,2018-08-01 01:55:29.000000000,2018-08-10 17:26:28.000000000,closed,1,Exit code for Pytype run with errors found
107,2018-07-31 02:12:44.000000000,2018-08-04 01:29:37.000000000,closed,3,Test PR Please Ignore
106,2018-07-30 11:22:48.000000000,2018-07-30 22:21:27.000000000,closed,0,Make pytype/patches directory a toplevel directory.
105,2018-07-29 10:16:38.000000000,2018-09-27 05:06:42.000000000,closed,9,_type_ 'v' not supported
104,2018-07-29 08:47:19.000000000,2018-07-30 22:20:00.000000000,closed,0,Add a script to generate Python executables.     
103,2018-07-27 09:40:54.000000000,2018-07-27 17:44:53.000000000,closed,1,Google sync
102,2018-07-26 14:57:07.000000000,2018-07-27 04:56:27.000000000,closed,2,Google sync
101,2018-07-20 23:15:26.000000000,2018-07-20 23:43:18.000000000,closed,2,Google sync
100,2018-07-18 23:45:07.000000000,2018-07-19 02:06:01.000000000,closed,0,"Log cmake and ninja outputs, and add more diagnostic messages to stdout."
99,2018-07-18 09:12:14.000000000,2018-07-18 22:59:02.000000000,closed,1,Google sync
98,2018-07-18 03:40:24.000000000,2018-07-18 06:21:32.000000000,closed,4,Google sync
97,2018-07-18 02:14:13.000000000,2018-07-18 02:39:04.000000000,closed,1,Google sync
96,2018-07-18 00:14:27.000000000,2018-07-18 00:42:05.000000000,closed,1,Install python3.5-dev and python3.6-dev.
95,2018-07-14 07:44:10.000000000,2018-07-18 02:07:24.000000000,closed,14,Google sync
94,2018-07-02 09:15:00.000000000,2018-10-05 01:37:42.000000000,closed,1,Importing pyi files with multi-level absolute imports causes a SyntaxError
93,2018-06-30 00:26:55.000000000,2018-06-30 01:14:18.000000000,closed,4,Project import generated by Copybara.
92,2018-06-28 09:17:46.000000000,1970-01-01 00:00:00.000000001,open,6,Output errors in machine-friendly format?
91,2018-06-21 22:28:18.000000000,2018-06-21 23:46:13.000000000,closed,4,Report import errors due to bad deps.
90,2018-06-19 02:51:56.000000000,1970-01-01 00:00:00.000000001,open,2,PATH issues when running in windows
89,2018-06-15 01:46:24.000000000,1970-01-01 00:00:00.000000001,open,2,Add support for protocol buffers
88,2018-06-14 22:42:26.000000000,2018-06-14 23:49:31.000000000,closed,0,Update typeshed.
87,2018-05-23 03:25:31.000000000,2018-06-05 03:25:08.000000000,closed,2,Imports Failing in Virtualenv
86,2018-05-18 10:43:36.000000000,2018-05-18 10:47:04.000000000,closed,0,Update typeshed submodule to 6a080cd0db8fe800fd3cae8769fd5320a02cc7c6
85,2018-05-18 10:18:42.000000000,2018-05-18 10:35:09.000000000,closed,3,Update typeshed submodule.
84,2018-05-13 07:12:12.000000000,2018-05-15 07:10:54.000000000,closed,2,Failing with a CompileError due to `-S` being passed to the compilation interpreter
83,2018-05-03 04:13:31.000000000,2018-05-04 00:01:53.000000000,closed,3,pytype scripts can no longer be run globally
82,2018-04-28 08:27:00.000000000,2018-05-01 01:14:08.000000000,closed,3,Clarification for the example of checking __init__.py in README
81,2018-04-28 07:10:02.000000000,1970-01-01 00:00:00.000000001,open,1,Is there a way to check return type error incompatible with supertype?
80,2018-04-28 04:57:32.000000000,2018-04-28 05:28:33.000000000,closed,5,How can pytype handle packages not compatible with pytype
79,2018-04-25 21:24:07.000000000,2018-08-11 09:33:56.000000000,closed,3,No extension definition for new C++ extension version of pytype.typegraph.cfg
78,2018-04-22 07:33:02.000000000,2018-05-16 01:01:22.000000000,closed,3,setup.py install Assertion Error with Python 3.6.5
77,2018-03-15 03:49:27.000000000,2019-05-16 07:17:40.000000000,closed,1,"open('foo', 'rb') treated as a file of str rather than bytes in Python 3"
76,2018-03-14 04:51:49.000000000,2018-09-27 01:34:16.000000000,closed,3,Invalid error from argparse.ArgumentParser.add_argument
75,2018-03-13 09:28:12.000000000,2018-09-27 03:42:14.000000000,closed,2,None return type generates error message
74,2018-02-08 23:52:03.000000000,2018-10-24 00:08:45.000000000,closed,1,Fix documentation: change .pytd to .pyi
73,2018-02-08 07:46:44.000000000,2019-03-19 00:52:25.000000000,closed,7,TypeVar doesn't work with quoted bound type
72,2018-01-29 10:02:18.000000000,2019-01-30 04:34:26.000000000,closed,7,pytype can't handle packages
71,2018-01-28 21:50:08.000000000,2018-07-02 08:33:33.000000000,closed,0,Make the parser_ext extension compile under python3
70,2017-12-23 17:31:09.000000000,2017-12-23 19:38:01.000000000,closed,5,typeshed tests are broken due to a recent pytype commit
69,2017-12-13 10:41:35.000000000,2017-12-14 04:12:38.000000000,closed,4,typeshed downgraded instead of upgraded
68,2017-12-10 15:27:33.000000000,2018-08-07 02:07:04.000000000,closed,4,Fix file path for assertion in setup.py
67,2017-12-05 04:06:24.000000000,2018-08-07 23:07:04.000000000,closed,3,"Fix repeated ""the"" in CONTRIBUTING.md"
66,2017-10-26 07:24:45.000000000,2018-05-16 01:00:49.000000000,closed,1,PyPI availability
65,2017-09-21 01:41:19.000000000,2018-02-07 04:17:59.000000000,closed,2,Support for multiple files and following import statements?
64,2017-09-09 05:05:37.000000000,2017-09-09 05:40:50.000000000,closed,1,Allow installing in virtualenv
63,2017-08-24 04:05:49.000000000,2018-02-07 04:19:26.000000000,closed,2,AttributeError on bare raise? (Python3 code being analyzed)
62,2017-08-12 07:01:18.000000000,2019-03-13 05:13:39.000000000,closed,5,Incorrectly inferring result of ternary operation with lambda
61,2017-06-21 19:39:00.000000000,2017-06-21 21:33:42.000000000,closed,1,typeshed fails on builtin
60,2017-06-13 20:23:23.000000000,2018-08-11 09:37:05.000000000,closed,3,PyType fails to parse `six.pyi` from Typeshed
59,2017-05-24 01:42:05.000000000,2018-09-27 02:45:08.000000000,closed,5,Parser crash on a generic type alias
58,2017-05-21 02:04:25.000000000,1970-01-01 00:00:00.000000001,open,3,Improve inference by enabling `--protocols` by default.
57,2017-04-01 11:06:49.000000000,2017-04-01 11:07:18.000000000,closed,0,Fix instructions in README.md
56,2017-03-31 15:22:49.000000000,2017-05-16 03:12:21.000000000,closed,2,Always crashes with '--python_version=3.5'
55,2017-03-31 15:00:49.000000000,2018-05-16 01:00:26.000000000,closed,1,No tags or releases
54,2017-03-29 14:07:16.000000000,2017-03-29 18:30:39.000000000,closed,3,pytype cannot analyze Python 3: invalid syntax [python-compiler-error]
53,2017-03-19 01:31:10.000000000,2017-04-22 02:25:18.000000000,closed,6,Incorrect ParseError for type comment in unexpected location
52,2017-03-14 08:56:09.000000000,2017-06-20 22:32:27.000000000,closed,0,fix installation instructions
51,2017-02-20 20:47:12.000000000,2017-02-20 21:11:46.000000000,closed,5,Current master fails to pip install
50,2017-02-16 02:30:40.000000000,1970-01-01 00:00:00.000000001,open,7,pytype doesn't seem to support nested classes
49,2017-02-15 21:14:29.000000000,2020-04-28 02:25:58.000000000,closed,2,"Please support ""long form"" per-parameter type comments"
48,2017-02-15 05:11:48.000000000,2017-02-20 21:15:26.000000000,closed,6,Support loading of 3rd party stubs from typeshed.
47,2017-02-14 04:23:17.000000000,2017-06-20 02:10:12.000000000,closed,8,Output may contain invalid import syntax when input file has leading ./
46,2017-02-10 20:01:03.000000000,2017-02-10 20:08:35.000000000,closed,7,typeshed/stdlib/2.7 has been moved to .../2.
45,2017-02-06 13:01:00.000000000,2017-02-09 20:34:11.000000000,closed,3,Error with unittest.pyi (typeshed too old?)
44,2017-02-06 12:33:14.000000000,2017-02-06 23:42:40.000000000,closed,0,Use an official Github URL in .gitmodules
43,2017-01-23 05:43:20.000000000,2017-04-25 03:18:13.000000000,closed,2,Pytype crashes when analysing `with open(..) as f` in python3.5
42,2017-01-18 21:53:50.000000000,2018-09-08 05:12:49.000000000,closed,2,pytype crashes with syntax error on del statement
41,2016-10-17 21:51:15.000000000,2017-05-16 03:36:01.000000000,closed,1,Crash when analyzing f-strings
40,2016-10-13 00:39:42.000000000,2017-05-16 03:14:41.000000000,closed,1,Crashes with Python 3.5 script
39,2016-10-12 14:45:38.000000000,2016-10-12 20:31:57.000000000,closed,1,Cannot handle Python 3 print()
38,2016-10-08 08:50:00.000000000,2017-05-16 03:32:39.000000000,closed,2,pytype doesn't support opcode 82
37,2016-09-08 19:43:39.000000000,2017-05-16 03:51:33.000000000,closed,1,pytype crashes when trying to generate stubs for `werkzeug`
36,2016-09-05 19:46:14.000000000,2017-05-16 03:23:50.000000000,closed,2,Crash with the attached script
35,2016-08-11 12:34:25.000000000,2017-05-16 03:16:34.000000000,closed,1,Support `elif` in conditionnals
34,2016-08-02 21:06:54.000000000,2018-03-15 00:30:47.000000000,closed,1,Fail to parse TypeVar bound with Generic
33,2016-08-02 17:29:09.000000000,2018-09-08 05:06:14.000000000,closed,2,"Incompatible templates, Name 'typing.AnyStr' must be defined as a TypeVar"
32,2016-08-02 01:37:15.000000000,2016-08-02 17:00:08.000000000,closed,3,Support @abstractmethod
31,2016-07-30 01:17:37.000000000,2018-09-08 04:54:34.000000000,closed,3,pytype spins CPU when trying to generate stubs for `relativedelta.py `
30,2016-07-26 12:31:55.000000000,2016-08-02 17:01:25.000000000,closed,2,Unable to parse varargs with empty name (Keyword-Only Arguments)
29,2016-07-26 10:57:27.000000000,2016-07-29 05:22:48.000000000,closed,2,Unable to parse stub platform conditionals
28,2016-07-15 12:12:38.000000000,2016-07-15 20:59:21.000000000,closed,1,Unable to parse stub conditionals
27,2016-07-15 11:54:32.000000000,2017-02-20 21:14:06.000000000,closed,2,add pytype.pytd.builtins to setup.py
26,2016-06-09 15:55:22.000000000,2018-05-16 00:57:20.000000000,closed,8,Release on PyPI
25,2016-06-06 18:57:41.000000000,2018-09-27 02:52:44.000000000,closed,32,Python 3 support?
24,2016-06-01 05:26:18.000000000,2018-10-09 22:19:00.000000000,closed,1,Document some types in __builtins__.
23,2016-06-01 03:30:23.000000000,2016-08-02 17:00:47.000000000,closed,1,mercurial.util also gives pytype a sad
22,2016-06-01 03:29:18.000000000,2016-07-21 22:03:48.000000000,closed,3,mercurial.error gives pytype a sad
21,2016-05-14 02:27:34.000000000,2018-04-03 00:46:19.000000000,closed,20,Generating pyis in the right order is cumbersome
20,2016-01-30 14:39:48.000000000,2016-01-30 19:41:53.000000000,closed,1,Make pytype installable via pip
19,2016-01-28 09:28:39.000000000,2016-03-01 23:25:47.000000000,closed,1,wrong type of 'self'
18,2016-01-28 08:37:57.000000000,2016-03-01 23:26:03.000000000,closed,1,optional parameters
17,2016-01-26 03:27:33.000000000,2016-03-01 23:26:13.000000000,closed,1, Make pytype understand typing.Callable
16,2016-01-26 03:25:10.000000000,1970-01-01 00:00:00.000000001,open,3,Use typeshed for (the remaining files in) pytd/builtins/*.pytd
15,2016-01-26 01:06:40.000000000,2016-03-01 23:25:29.000000000,closed,0,invalid type parameter matches
14,2016-01-23 05:08:09.000000000,2016-01-26 03:19:06.000000000,closed,7,"Traceback: ""ValueError: Unresolved class: 'False'"""
13,2016-01-22 22:31:53.000000000,2016-06-01 07:18:31.000000000,closed,1,pyi is missing imports
12,2016-01-22 07:23:09.000000000,2016-01-23 03:23:02.000000000,closed,1,"PEP 484 says to use None, not NoneType"
11,2016-01-22 01:30:43.000000000,2016-01-27 01:29:48.000000000,closed,3,Unify union of a class with a subclass of it?
10,2016-01-22 01:22:31.000000000,2018-09-27 02:55:34.000000000,closed,12,"Dict[slice, ...] is not valid"
9,2016-01-22 01:18:48.000000000,2016-01-26 03:11:51.000000000,closed,1,"Should use List[T] instead of List[T, ...]"
8,2016-01-21 23:14:56.000000000,2016-01-28 23:14:29.000000000,closed,2,How does -c work?
7,2016-01-21 09:28:44.000000000,2016-03-17 18:02:46.000000000,closed,0,duplicate signature
6,2016-01-21 07:57:52.000000000,2016-01-31 00:51:33.000000000,closed,5,python -m pip install -U . doesn't work.
5,2016-01-21 07:44:08.000000000,2016-05-17 03:49:49.000000000,closed,2,Python 3 compatibility needs work
4,2016-01-21 07:34:35.000000000,2016-01-23 03:23:58.000000000,closed,5,pytype generates invalid pyi
3,2015-11-17 20:18:12.000000000,2015-12-05 04:03:26.000000000,closed,1,ImportError: No module named 'StringIO'
2,2015-11-03 11:34:43.000000000,2015-11-07 09:54:46.000000000,closed,2,Project status and plans
1,2015-10-18 13:44:40.000000000,2016-01-22 03:13:20.000000000,closed,2,Really requires PLY.
